CrazyHand version 1.0/src/com/FileIO.java

```diff
@@ -614,7 +614,11 @@ public static void save() {
 		if(MeleeEdit.selectedMenu==MeleeEdit.MENU_ATTRIBUTES){
 			f.position(Character.characters[MeleeEdit.selected].offset);
 			for (int i = 0; i < Attribute.attributes.length; i++) {
-				f.putFloat((float) MeleeEdit.attributeTable.getValueAt(i, 1));
+				if(i != 22){ // Index for "Number of Jumps" attribute
+					f.putFloat((float) MeleeEdit.attributeTable.getValueAt(i, 1));
+				}else{
+					f.putInt(Math.round((float) MeleeEdit.attributeTable.getValueAt(i, 1)));

```

Comment:
Without the Math.round the save fails as getValueAt (in this case) returns a float.